opensearch_py_ml/ml_commons/rag_pipeline/rag/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ignore data and ingestion directories
```

Comment:
can't we use this file: https://github.com/opensearch-project/opensearch-py-ml/blob/main/.gitignore?

Reply:
sure we can, let me change.